### ISSUE TITLE
'could not find function phewas_meta_logic'

### ISSUE THREAD

**Issue by aeronsmall** [PheWAS/PheWAS#19]
Hello - I am having trouble running the phewasMeta() function. I get the following error message:

Error in phewas_meta_logic(., fixed = fixed, ...) : 
  could not find function "phewas_meta_logic"

Let me know if there is a workaround.

Best,

AMS

**Comment by RobertJCarroll** [comment]
It sounds like you have a namespace issue. Is it possible you have a modified copy of the phewasMeta() in your environment? The code doesn't specify the namespace of phewas_meta_logic, so if one were to source the phewasMeta.R script it would yield the error you are observing.

If that's not the case, I would try a couple of things:

1. Load the package using library(PheWAS) (May not be necessary, but might help debug)
2. Re-install the package (I just specified the the namespace, so might help either way).

If that doesn't work, can you provide some more context, particularly the sessionInfo() output? The command that you ran would also be helpful.

**Comment by aeronsmall** [comment]
Thank you for the response. I am unable to run the example provided results.meta=phewasMeta(results, fixed=FALSE, keep.both=FALSE)

My session.info() is: 

R version 3.5.1 (2018-07-02)
Platform: x86_64-apple-darwin15.6.0 (64-bit)
Running under: macOS Sierra 10.12.3

Matrix products: default
BLAS: /System/Library/Frameworks/Accelerate.framework/Versions/A/Frameworks/vecLib.framework/Versions/A/libBLAS.dylib
LAPACK: /Library/Frameworks/R.framework/Versions/3.5/Resources/lib/libRlapack.dylib

attached base packages:
[1] parallel  stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] PheWAS_0.12.2 ggplot2_3.1.0 tidyr_0.8.2   dplyr_0.7.7  

loaded via a namespace (and not attached):
 [1] Rcpp_0.12.19     nloptr_1.2.1     pillar_1.3.0     compiler_3.5.1   plyr_1.8.4       bindr_0.1.1     
 [7] tools_3.5.1      rpart_4.1-13     lme4_1.1-18-1    digest_0.6.18    nlme_3.1-137     tibble_1.4.2    
[13] gtable_0.2.0     lattice_0.20-35  mgcv_1.8-24      logistf_1.23     pkgconfig_2.0.2  rlang_0.3.0.1   
[19] Matrix_1.2-14    rstudioapi_0.8   ggrepel_0.8.0    bindrcpp_0.2.2   withr_2.1.2      htmlwidgets_1.3 
[25] lmtest_0.9-36    nnet_7.3-12      grid_3.5.1       DT_0.4           tidyselect_0.2.5 mice_3.3.0      
[31] glue_1.3.0       R6_2.3.0         mitml_0.3-6      survival_2.42-3  minqa_1.2.4      meta_4.9-2      
[37] purrr_0.2.5      magrittr_1.5     backports_1.1.2  scales_1.0.0     htmltools_0.3.6  MASS_7.3-50     
[43] splines_3.5.1    assertthat_0.2.0 colorspace_1.3-2 lazyeval_0.2.1   munsell_0.5.0    broom_0.5.0     
[49] pan_1.6          crayon_1.3.4     zoo_1.8-4        jomo_2.6-4

**Comment by RobertJCarroll** [comment]
I've resolved the problem- I didn't realize the .Rbuildignore was detailed regular expressions, so it was overmatching and removing some function files under some build conditions. Thank you for bringing this to my attention!

**Comment by aeronsmall** [comment]
Thanks! Still having trouble, however. Even with installing from source I now get the following message:

Error: 'phewas_meta_logic' is not an exported object from 'namespace:PheWAS'

Any idea how to troubleshoot?

**Comment by RobertJCarroll** [comment]
Thanks. I altered an original scoping change I applied as it gave me a warning in my R CMD check. This is the issue I expected originally: the dplyr::do call scope/namespace is different from the PheWAS::phewasMeta scope, so there were some namespace conflicts. I've updated this again, I've tested it to work now. I'll be more aware of this if I attempt to resolve the build warning in the future.
